File: app/src/main/java/in/woloo/www/dashboard/ui/period_reminder/PeriodReminderViewModel.java
package in.woloo.www.dashboard.ui.period_reminder;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class PeriodReminderViewModel extends ViewModel {

    private MutableLiveData<String> mText;

    public PeriodReminderViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("This is gallery fragment");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: app/src/main/java/in/woloo/www/more/period_reminder/PeriodReminderViewModel.kt
package `in`.woloo.www.more.period_reminder

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class PeriodReminderViewModel : ViewModel() {
    private val mText =
        MutableLiveData<String>()

    init {
        mText.value = "This is gallery fragment"
    }

    val text: LiveData<String>
        get() = mText
}